eyeshade/workers/reports.js

```diff
@@ -910,7 +910,7 @@ const referralStatement = async (debug, runtime, owner, summaryP) => {
 const findEligPublishers = async (debug, runtime, publishers) => {
   const publishersC = runtime.database.get('publishers', debug)
 
-  if (!publishers) {
+  if (!publishers || publishers.length === 0) {
```

Comment:
fyi: i don't think that it is possible for the `pubishers` argument to ever be null/zero/undefined/etc., but empty is certainly possible...regardless,  it is good to have the check in the way you have it, as a safety.